indico/modules/events/abstracts/controllers/management.py

```diff
@@ -55,3 +60,36 @@ def _process(self):
             flash(_('Book of Abstract settings have been saved'), 'success')
             return jsonify_data()
         return jsonify_form(form)
+
+
+class RHAbstractList(RHManageAbstractsBase):
+    """Display the list of abstracts"""
+
+    def _process(self):
+        if self.list_generator.static_link_used:
+            return redirect(self.list_generator.get_list_url())
+        list_kwargs = self.list_generator.get_list_kwargs()
+        return WPManageAbstracts.render_template('management/abstract_list.html', self._conf, event=self.event_new,
+                                                 **list_kwargs)
+
+
+class RHAbstractListCustomize(RHManageAbstractsBase):
+    """Filter options and columns to display for the abstract list of an event"""
+
+    def _process_GET(self):
+        list_config = self.list_generator._get_config()
+        return WPManageAbstracts.render_template('management/abstract_list_filter.html', self._conf,
+                                                 event=self.event_new, visible_items=list_config['items'],
+                                                 static_items=self.list_generator.static_items,
+                                                 filters=list_config['filters'])
+
+    def _process_POST(self):
+        self.list_generator.store_configuration()
+        return jsonify_data(**self.list_generator.render_list())
```

Comment:
Do you ever flash something in `render_list` or `store_configuration`? If not, add `flash=False` to this call.